### PR TITLE
tags/instructor/feedbacks/*.tag: change to 2-space indentation #7352

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352 
Applied to all files in `src/main/webapp/WEB-INF/tags/instructor/feedbacks/*`

### CODE REVIEW

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
-   </c:if>
-   data-sending-published-email="${publishButton.sendingPublishedEmail}"
-   data-fsname="${publishButton.feedbackSessionName}"
-   <c:if test="${not publishButton.actionAllowed}">disabled</c:if>>
-    ${publishButton.actionName} Results
+    class="<c:if test="${not empty buttonType}">${buttonType} </c:if>session-${publishButton.actionNameLowercase}-for-test"
+    <c:if test="${showTooltip}">
+        title="${publishButton.tooltipText}"
+        data-toggle="tooltip"
+        data-placement="top"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag]: Too much indentation.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 2]
```diff
@@ -8,274 +8,277 @@
...
+                        id="<%= Const.ParamsNames.COURSE_ID %>">
+                      <c:forEach items="${fsForm.coursesSelectField}" var="option">
+                        <option ${option.attributesToString}>${option.content}</option>
+                      </c:forEach>
+                    </select>
+                  </c:when>
+                  <c:otherwise>
+                      <h5 class="form-control-static font-weight-normal">
+                        ${fsForm.courseId}
+                      </h5>
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Remove 2 spaces from this `h5` element.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 3]
```diff
@@ -8,274 +8,277 @@
...
+                        name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>"
+                        id="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>"
+                        maxlength="<%= FieldValidator.FEEDBACK_SESSION_NAME_MAX_LENGTH %>"
+                        placeholder="e.g. Feedback for Project Presentation 1"
+                        value="${fsForm.fsName}">
+                  </c:when>
+                  <c:otherwise>
+                      <h5 class="form-control-static font-weight-normal">
+                        ${fsForm.fsName}
+                      </h5>
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Remove 2 spaces from this `h5` element too.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag [hunk 4]
```diff
@@ -6,242 +6,231 @@
...
+        <div class="row">
+          <div class="col-xs-12"
+              title="<%= Const.Tooltips.FEEDBACK_SESSION_RESULTSVISIBLELABEL %>"
+              data-toggle="tooltip"
+              data-placement="top">
+            <label class="label-control">Responses visible from</label>
+          </div>
+        </div>
+        <div class="row radio"><%
+          %>
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag]: It should be safe to remove the `<% %>` here, although this isn't part of the issue.

**Comment by sujeet14108** [comment]
Done

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsTable.tag [hunk 5]
```diff
@@ -7,60 +7,58 @@
...
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                </tr>
-            </c:otherwise>
-        </c:choose>
+      </c:otherwise>
+    </c:choose>
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsTable.tag]: Missed this earlier: remove 2 spaces from this entire choose block.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/copyFromModal.tag [hunk 6]
```diff
@@ -6,83 +6,83 @@
...
-            <div class="modal-footer margin-0">
-                <button type="button" class="btn btn-primary" id="button_copy_submit" disabled>
-                    Copy
-                </button>
-                <button type="button" class="btn btn-default" data-dismiss="modal">
-                    Cancel
-                </button>
-            </div>
-        </div>
+  aria-labelledby="copyModalTitle" aria-hidden="true">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/copyFromModal.tag]: This needs more indentation.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/copyFromModal.tag [hunk 7]
```diff
@@ -6,83 +6,83 @@
...
+                name="<%= Const.ParamsNames.COPIED_FEEDBACK_SESSION_NAME %>"
+                id="modalCopiedSessionName"
+                maxlength=<%= FieldValidator.FEEDBACK_SESSION_NAME_MAX_LENGTH %>
+                value="${copyFromModal.fsName}"
+                placeholder="e.g. Feedback for Project Presentation 1">
+          </div>
+          <%-- Previous Session --%>
+          <label>Copy sessions/questions from</label>
+          <table class="table-responsive table table-bordered table-hover margin-0"
+              id="copyTableModal">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/copyFromModal.tag]: Might as well combine this with the previous line.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsTable.tag [hunk 8]
```diff
@@ -7,60 +7,58 @@
...
+      <th id="button_sortid" class="button-sort-ascending course-id-table-width toggle-sort">
+        Course ID <span class="icon-sort unsorted"></span>
+      </th>
+      <th id="button_sortname" class="button-sort-none session-name-table-width toggle-sort">
+        Session Name <span class="icon-sort unsorted"></span>
+      </th>
+      <th>Status</th>
+      <th>
+        <span title="<%= Const.Tooltips.FEEDBACK_SESSION_RESPONSE_RATE %>"
+            data-toggle="tooltip" data-placement="top">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsTable.tag]: Might as well combine this with the previous line.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 9]
```diff
@@ -8,274 +8,277 @@
...
+      <div class="panel-body">
+        <div class="row">
+          <div class="col-sm-12 col-md-6"
+              title="<%= Const.Tooltips.FEEDBACK_SESSION_COURSE %>"
+              data-toggle="tooltip"
+              data-placement="top">
+            <div class="form-group<c:if test="${fsForm.showNoCoursesMessage}"> has-error</c:if>">
+              <h5 class="col-sm-2 col-md-4">
+                <label class="control-label"
+                    for="<%= Const.ParamsNames.COURSE_ID %>">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Might as well combine this with the previous line.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 10]
```diff
@@ -8,274 +8,277 @@
...
+            </div>
+          </div>
+          <div class="col-sm-12 col-md-6 tablet-no-mobile-margin-top-20px"
+              title="<%= Const.Tooltips.FEEDBACK_SESSION_INPUT_TIMEZONE %>"
+              data-toggle="tooltip"
+              data-placement="top">
+            <div class="form-group">
+              <h5 class="col-sm-2 col-md-4">
+                <label class="control-label"
+                    for="<%= Const.ParamsNames.FEEDBACK_SESSION_TIMEZONE %>">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Same here.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 11]
```diff
@@ -8,274 +8,277 @@
...
+        <br class="hidden-xs">
+        <div class="row">
+          <div class="col-sm-12"
+              title="<%= Const.Tooltips.FEEDBACK_SESSION_INPUT_NAME %>"
+              data-toggle="tooltip"
+              data-placement="top">
+            <div class="form-group">
+              <h5 class="col-sm-2">
+                <label class="control-label"
+                    for="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Same here.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 12]
```diff
@@ -8,274 +8,277 @@
...
+        <br class="hidden-xs">
+        <div class="row" id="instructionsRow">
+          <div class="col-sm-12"
+              title="<%= Const.Tooltips.FEEDBACK_SESSION_INSTRUCTIONS %>"
+              data-toggle="tooltip"
+              data-placement="top">
+            <div class="form-group">
+              <h5 class="col-sm-2 margin-top-0">
+                <label class="control-label"
+                    for="<%= Const.ParamsNames.FEEDBACK_SESSION_INSTRUCTIONS %>">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Same here.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 13]
```diff
@@ -8,274 +8,277 @@
...
+      <div class="panel-body">
+        <div class="row">
+          <div class="col-md-5"
+              title="<%= Const.Tooltips.FEEDBACK_SESSION_STARTDATE %>"
+              data-toggle="tooltip"
+              data-placement="top">
+            <div class="row">
+              <div class="col-xs-12">
+                <label class="label-control"
+                    for="<%= Const.ParamsNames.FEEDBACK_SESSION_STARTDATE %>">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Same here.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 14]
```diff
@@ -8,274 +8,277 @@
...
-                    <button id="button_submit" type="submit" class="btn btn-primary"
-                        <c:if test="${fsForm.submitButtonDisabled}">disabled</c:if>
-                        <c:if test="${!fsForm.submitButtonVisible}"> style="display:none;" </c:if>
-                    >
-                            ${fsForm.submitButtonText}
-                    </button>
-                </div>
+              <div class="col-xs-12">
+                <label class="label-control"
+                    for="<%= Const.ParamsNames.FEEDBACK_SESSION_ENDDATE %>">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Same here.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 15]
```diff
@@ -8,274 +8,277 @@
...
+            </div>
+          </div>
+          <div class="col-md-2 border-left-gray"
+              title="<%= Const.Tooltips.FEEDBACK_SESSION_INPUT_GRACEPERIOD %>"
+              data-toggle="tooltip"
+              data-placement="top">
+            <div class="row">
+              <div class="col-xs-12">
+                <label class="control-label"
+                    for="<%= Const.ParamsNames.FEEDBACK_SESSION_GRACEPERIOD %>">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Same here.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag [hunk 16]
```diff
@@ -8,274 +8,277 @@
...
+    </c:if>
+    <c:if test="${!fsForm.fsNameEditable}">
+      <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>" value="${fsForm.fsName}">
+    </c:if>
+    <c:if test="${!fsForm.courseIdEditable}">
+      <input type="hidden" name="<%= Const.ParamsNames.COURSE_ID %>" value="${fsForm.courseId}">
+    </c:if>
+    <input type="hidden"
+        name="<%= Const.ParamsNames.USER_ID %>"
+        value="${data.account.googleId}">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag]: Same here.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag [hunk 17]
```diff
@@ -6,242 +6,230 @@
...
+            <label for="<%= Const.ParamsNames.FEEDBACK_SESSION_RESULTSVISIBLEBUTTON %>_never">
+              Never
+            </label>
+            <input type="radio"
+                name="<%= Const.ParamsNames.FEEDBACK_SESSION_RESULTSVISIBLEBUTTON %>"
+                id="<%= Const.ParamsNames.FEEDBACK_SESSION_RESULTSVISIBLEBUTTON %>_never"
+                value="<%= Const.INSTRUCTOR_FEEDBACK_RESULTS_VISIBLE_TIME_NEVER %>"
+                <c:if test="${additionalSettings.responseVisibleNeverChecked}">checked</c:if>>
+          </div>
+         </div>
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag]: One space to the left.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag [hunk 18]
```diff
@@ -6,242 +6,230 @@
...
+      </div>
     </div>
+  </div>
 </div>
  <div class="panel panel-primary" style="display:none;" id="sendEmailsForPanel">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag]: Move the whole block one space to the left.

File: src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag [hunk 19]
```diff
@@ -6,242 +6,230 @@
...
-            </div>
+  <div class="panel-body">
+    <div class="row">
+      <div class="col-md-12">
+        <label class="control-label">Send emails for</label>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-3"
+        title="<%= Const.Tooltips.FEEDBACK_SESSION_SENDOPENEMAIL %>"
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag]: Needs more indentation.

Comment (sujeet14108) [inline src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag]: Done

**Pull request merged**